### PR TITLE
Update pricing text EN

### PR DESCRIPTION
**Pull request by LenaTovar**
Same inputs as in DE.
Please double check the EN text as I am no native writer.

### CODE REVIEW

File: app/config/translations/text_en.yml [hunk 1]
```diff
@@ -88,8 +88,8 @@ text:
       link: https://rokka.io/assets/pdf/Rokka_Terms_of_use_EN.pdf
 
   cta:
-    title: Try out rokka for 90 days.
-    text: Follow companies like Migros or FREITAG – start now with rokka.
+    title: Every start should be free. Try out rokka without any risk.
```
[the remark this answers is not in the document]
Comment (LenaTovar) [reply app/config/translations/text_en.yml]: Is fine for me as well.

File: app/config/translations/text_en.yml [hunk 2]
```diff
@@ -56,18 +56,18 @@ text:
...
       monthly: per month
       storage_text1: per 1 GB of storage
       storage_text2: up to 400 GB
       traffic_text1: per 1 GB of traffic
       traffic_text2: up to 2000 GB per month
     details:
-      title: Pay less when using more than 400 GB storage or 2000 GB traffic.
-      text: See details
+      title: You want even more? Pay less when using more. Profit from our
+      text: keen conditions.
```
[the remark this answers is not in the document]
Comment (LenaTovar) [reply app/config/translations/text_en.yml]: Merlin confirmed "attractive rates". I changed it already.

File: app/config/translations/text_en.yml [hunk 3]
```diff
@@ -77,7 +77,7 @@ text:
       contact: Contact us for an individual offer
       owncdn:  Have your own CDN?
     free:
-      title: Free plan
+      title: Test for free
```
Comment (dbu) [inline app/config/translations/text_en.yml]: this is not only for testing but a full free plan that you can try out.

**Comment by dbu** [comment]
thanks a lot for the updates!

**Pull request merged**